crates/bevy_ui/src/widget/text.rs

```diff
@@ -22,14 +24,26 @@ fn scale_value(value: f32, factor: f64) -> f32 {
 
 /// Defines how `min_size`, `size`, and `max_size` affects the bounds of a text
 /// block.
-pub fn text_constraint(min_size: Val, size: Val, max_size: Val, scale_factor: f64) -> f32 {
+pub fn text_constraint(
+    min_size: Val,
+    size: Val,
+    max_size: Val,
+    scale_factor: f64,
+    container_size: f32,
+) -> f32 {
     // Needs support for percentages
```

Comment:
This comment can probably be removed since you're implementing the support for percentages here.